#goal definition
geometry_msgs/Pose pose_goal
---
#result definition
geometry_msgs/Pose pose_final
---
#feedback
geometry_msgs/Pose pose_current
